build/make-stacks.R

```diff
@@ -110,7 +110,7 @@ library(gert)
   return(.url)
 }
 
-.cuda_baseimage_tag <- function(ubuntu_series, other_variants = "11.1.1-cudnn8-devel") {
+.cuda_baseimage_tag <- function(ubuntu_series, other_variants = "11.7.1-cudnn8-devel") {
```

Comment:
It may be worthwhile to automatically update the cuda version instead of hard-coding it.